frontend/src/helpers/index.ts

```diff
@@ -358,7 +358,9 @@ function loadTableSettings(tableId: string): { id: string; show: boolean }[] {
  *
  * The app also passes the token to the headlamp-server via HEADLAMP_BACKEND_TOKEN env var.
  */
-const backendToken = new URLSearchParams(window.location.search).get('backendToken');
+const backendToken =
+  process.env.REACT_APP_HEADLAMP_BACKEND_TOKEN ||
```

Comment:
Hrmm.

This puts the token into the compiled javascript,
so maybe it's not so safe to do it this way.

I'm trying to think through all the cases where someone might set that variable. Maybe it's ok.

Reply:
Can you explain how this puts the token into the compiled js?